packages/vapor/resources/icons/svg/associate.svg

```diff
@@ -0,0 +1,7 @@
+<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M12.5 6.28564V5.78564H11.5V6.28564H12.5ZM11.5 11.2856C11.5 11.5618 11.7239 11.7856 12 11.7856C12.2762 11.7856 12.5 11.5618 12.5 11.2856H11.5ZM13.8345 13.6283C13.6292 13.4435 13.3131 13.4602 13.1284 13.6654C12.9436 13.8707 12.9603 14.1868 13.1655 14.3716L13.8345 13.6283ZM18.1655 18.8716L18.5372 19.2061L19.2061 18.4628L18.8345 18.1283L18.1655 18.8716ZM10.8331 14.3728C11.039 14.1889 11.0569 13.8728 10.8729 13.6669C10.689 13.4609 10.3729 13.4431 10.1669 13.627L10.8331 14.3728ZM5.16366 18.13L4.79369 18.4663L5.46637 19.2062L5.83634 18.8699L5.16366 18.13ZM11.5 6.28564V11.2856H12.5V6.28564H11.5ZM13.1655 14.3716L18.1655 18.8716L18.8345 18.1283L13.8345 13.6283L13.1655 14.3716ZM10.5 13.9999C10.1669 13.627 10.1669 13.627 10.1669 13.627C10.1669 13.6271 10.1668 13.6271 10.1668 13.6271C10.1667 13.6272 10.1666 13.6273 10.1664 13.6275C10.1661 13.6278 10.1655 13.6283 10.1648 13.6289C10.1634 13.6301 10.1614 13.632 10.1586 13.6344C10.1532 13.6393 10.145 13.6466 10.1343 13.6562C10.1129 13.6753 10.0811 13.7037 10.04 13.7404C9.95777 13.814 9.8379 13.9211 9.68736 14.0559C9.38627 14.3253 8.96245 14.7049 8.47149 15.1456C7.48971 16.0267 6.23878 17.1526 5.16366 18.13L5.83634 18.8699C6.9091 17.8947 8.15818 16.7705 9.13943 15.8898C9.62997 15.4495 10.0534 15.0702 10.3542 14.801C10.5046 14.6664 10.6244 14.5594 10.7065 14.486C10.7476 14.4493 10.7792 14.421 10.8006 14.4019C10.8113 14.3923 10.8194 14.385 10.8249 14.3802C10.8276 14.3777 10.8296 14.3759 10.831 14.3747C10.8317 14.3741 10.8322 14.3736 10.8326 14.3733C10.8327 14.3731 10.8329 14.373 10.8329 14.3729C10.833 14.3729 10.833 14.3729 10.833 14.3729C10.8331 14.3728 10.8331 14.3728 10.5 13.9999Z" fill="#282829" />
+    <path d="M5.5 20C5.5 20.8284 4.82843 21.5 4 21.5C3.17157 21.5 2.5 20.8284 2.5 20C2.5 19.1716 3.17157 18.5 4 18.5C4.82843 18.5 5.5 19.1716 5.5 20Z" stroke="#282829" stroke-linecap="round" stroke-linejoin="round" />
+    <path d="M21.5 20C21.5 20.8284 20.8284 21.5 20 21.5C19.1716 21.5 18.5 20.8284 18.5 20C18.5 19.1716 19.1716 18.5 20 18.5C20.8284 18.5 21.5 19.1716 21.5 20Z" stroke="#282829" stroke-linecap="round" stroke-linejoin="round" />
+    <path d="M13.5 13C13.5 13.8284 12.8284 14.5 12 14.5C11.1716 14.5 10.5 13.8284 10.5 13C10.5 12.1716 11.1716 11.5 12 11.5C12.8284 11.5 13.5 12.1716 13.5 13Z" stroke="#282829" stroke-linecap="round" stroke-linejoin="round" />
+    <path d="M13.5 4C13.5 4.82843 12.8284 5.5 12 5.5C11.1716 5.5 10.5 4.82843 10.5 4C10.5 3.17157 11.1716 2.5 12 2.5C12.8284 2.5 13.5 3.17157 13.5 4Z" stroke="#282829" stroke-linecap="round" stroke-linejoin="round" />
+</svg>
```

Comment:
Stroke colour must only be on the root, otherwise we won't be able to easily override this colour with css.

```suggestion
<svg viewBox="0 0 24 24" stroke="#282829" fill="none" xmlns="http://www.w3.org/2000/svg">
    <path d="M12.5 6.28564V5.78564H11.5V6.28564H12.5ZM11.5 11.2856C11.5 11.5618 11.7239 11.7856 12 11.7856C12.2762 11.7856 12.5 11.5618 12.5 11.2856H11.5ZM13.8345 13.6283C13.6292 13.4435 13.3131 13.4602 13.1284 13.6654C12.9436 13.8707 12.9603 14.1868 13.1655 14.3716L13.8345 13.6283ZM18.1655 18.8716L18.5372 19.2061L19.2061 18.4628L18.8345 18.1283L18.1655 18.8716ZM10.8331 14.3728C11.039 14.1889 11.0569 13.8728 10.8729 13.6669C10.689 13.4609 10.3729 13.4431 10.1669 13.627L10.8331 14.3728ZM5.16366 18.13L4.79369 18.4663L5.46637 19.2062L5.83634 18.8699L5.16366 18.13ZM11.5 6.28564V11.2856H12.5V6.28564H11.5ZM13.1655 14.3716L18.1655 18.8716L18.8345 18.1283L13.8345 13.6283L13.1655 14.3716ZM10.5 13.9999C10.1669 13.627 10.1669 13.627 10.1669 13.627C10.1669 13.6271 10.1668 13.6271 10.1668 13.6271C10.1667 13.6272 10.1666 13.6273 10.1664 13.6275C10.1661 13.6278 10.1655 13.6283 10.1648 13.6289C10.1634 13.6301 10.1614 13.632 10.1586 13.6344C10.1532 13.6393 10.145 13.6466 10.1343 13.6562C10.1129 13.6753 10.0811 13.7037 10.04 13.7404C9.95777 13.814 9.8379 13.9211 9.68736 14.0559C9.38627 14.3253 8.96245 14.7049 8.47149 15.1456C7.48971 16.0267 6.23878 17.1526 5.16366 18.13L5.83634 18.8699C6.9091 17.8947 8.15818 16.7705 9.13943 15.8898C9.62997 15.4495 10.0534 15.0702 10.3542 14.801C10.5046 14.6664 10.6244 14.5594 10.7065 14.486C10.7476 14.4493 10.7792 14.421 10.8006 14.4019C10.8113 14.3923 10.8194 14.385 10.8249 14.3802C10.8276 14.3777 10.8296 14.3759 10.831 14.3747C10.8317 14.3741 10.8322 14.3736 10.8326 14.3733C10.8327 14.3731 10.8329 14.373 10.8329 14.3729C10.833 14.3729 10.833 14.3729 10.833 14.3729C10.8331 14.3728 10.8331 14.3728 10.5 13.9999Z" fill="#282829" />
    <path d="M5.5 20C5.5 20.8284 4.82843 21.5 4 21.5C3.17157 21.5 2.5 20.8284 2.5 20C2.5 19.1716 3.17157 18.5 4 18.5C4.82843 18.5 5.5 19.1716 5.5 20Z"  stroke-linecap="round" stroke-linejoin="round" />
    <path d="M21.5 20C21.5 20.8284 20.8284 21.5 20 21.5C19.1716 21.5 18.5 20.8284 18.5 20C18.5 19.1716 19.1716 18.5 20 18.5C20.8284 18.5 21.5 19.1716 21.5 20Z"  stroke-linecap="round" stroke-linejoin="round" />
    <path d="M13.5 13C13.5 13.8284 12.8284 14.5 12 14.5C11.1716 14.5 10.5 13.8284 10.5 13C10.5 12.1716 11.1716 11.5 12 11.5C12.8284 11.5 13.5 12.1716 13.5 13Z"  stroke-linecap="round" stroke-linejoin="round" />
    <path d="M13.5 4C13.5 4.82843 12.8284 5.5 12 5.5C11.1716 5.5 10.5 4.82843 10.5 4C10.5 3.17157 11.1716 2.5 12 2.5C12.8284 2.5 13.5 3.17157 13.5 4Z"  stroke-linecap="round" stroke-linejoin="round" />
</svg> 
```

Reply:
Good to know!